src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -517,22 +517,26 @@ export function getStatusContents(statusType, details = {}, name = {}) {
 export const EVENT_TYPES = {
   claimDecision: 'claim_decision',
   nod: 'nod',
-  droHearing: 'dro_hearing',
   fieldGrant: 'field_grant',
   soc: 'soc',
   form9: 'form9',
   ssoc: 'ssoc',
   certified: 'certified',
   hearingHeld: 'hearing_held',
-  hearingCancelled: 'hearing_cancelled',
```

Comment:
Did you do a search for all these removed event types? I think it should be fine, but I just want to make sure.

Reply:
Also there is another [helper case for hearingCancelled](https://github.com/department-of-veterans-affairs/vets-website/blob/24f71a6be625b93f10f173c414eb22976b0ec3b0/src/js/claims-status/utils/appeal-helpers.jsx#L29) that should be removed in appealStatusDescriptions.

Reply:
Oh, good find in the API! Didn't know that was a thing.

@kreek Looks like this needs to be updated.

@cmgiven Are the event types all up-to-date in the [swagger docs](https://app.swaggerhub.com/apis/dsva-appeals/appeals-status/2.0.0#/)?

@cehsu I think we can leave it in `appealStatusDescriptions` because this is for the status type, not the event type. That does raise an interesting question, though... @cmgiven Is it still going to be a status type, or will it be removed from that list as well? I guess more broadly, do the status and event type enums directly correlate?

Reply:
All events types are up to date in swagger minus the merge status, which currently there is a ticket for.